protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3899,19 +3899,38 @@ message EstimatePositionRequest {
   int64 open_volume = 2 [(google.api.field_behavior) = REQUIRED];
   // Open and/or hypothetical orders.
   repeated OrderInfo orders = 3;
-  // Collateral available for the position. If provided then response will contain the liquidation price estimate.
-  // Collateral available needs to be provided scaled to asset decimal places. Liquidation price returned is scaled to asset decimal places by default.
-  optional string collateral_available = 4;
+  // Margin account balance. Needs to be provided scaled to asset decimal places.
+  string margin_account_balance = 4;
+  // General account balance. Needs to be provided scaled to asset decimal places.
+  string general_account_balance = 5;
+  // Order margin account balance. Needs to be provided scaled to asset decimal places.
+  string order_margin_account_balance = 6;
+  // Margin mode for the party, cross margin or isolated margin.
+  vega.MarginMode margin_mode = 7;
+  // Margin factor to be used along with isolated margin mode
+  optional string margin_factor = 8;
+  // Whether the estimated collateral increase should be assumed as available for liquidation price calculation in isolated margin mode.
```

Comment:
I don't understand this description - could you rephrase, or describe what this does and we can work on it together?

Reply:
Would that be clearer?
> Whether the estimated collateral increase should be included in available collateral for liquidation price calculation in isolated margin mode.

It's for deal ticket purposes: there the user is specifing an order but the extra collateral needed for that order is not yet in any of the margin accounts, however the user would most likely be interested in the liquidaiton price assuming the order gets posted and the appropriate additional collateral amount gets transfered.

When looking at the existing position and orders the liquidation price should be calculated only based on current level of margin accounts (main and order account) without any transfers even if the margin account balance is below the current initial margin level.

Reply:
That is clearer! Thank you for the explanation, too. Would you mind making that change for the GQL too?

Reply:
Sure, done now.